### PR TITLE
Drawlist additions

### PR DESCRIPTION
**Pull request by Berna-L**
* AddInputString, to set strings.
* AddIconButton, to use ImGuiComponent.IconButton with DrawList.
* StartConditional, so parts of the window can be shown only if a condition is met.
* EndConditional, to mark the end of the section dependant on the condition.

### CODE REVIEW

File: Drawing/DrawList.cs [hunk 1]
```diff
@@ -488,4 +508,53 @@ public T AddButton(string label, Action action, Vector2? buttonSize = null)
...
         return DrawListOwner;
     }
-}
\ No newline at end of file
+
+    public T AddIconButton(FontAwesomeIcon icon, Action action, string? hoverTooltip = null)
+    {
+        DrawActions.Add(() =>
+        {
+            if (ImGuiComponents.IconButton(icon))
```
Comment (MidoriKami) [inline Drawing/DrawList.cs]: Should use the overload for IconButton that takes a ID string, or else there will be an ID conflict if multiple of the same icon are used and only the first button will work.

**Comment by MidoriKami** [comment]
Can you remove these files from the commit: 
image

**Comment by Berna-L** [comment]
Updated the file list. Will follow your comment regarding the overload now.

File: Drawing/DrawList.cs (hunk 1 above)
Comment (Berna-L) [reply Drawing/DrawList.cs]: Is there a good way to define which ID to use? Should I just add an ID parameter to the signature of `AddIconButton`?

**Pull request merged**